src/idpi/operators/support_operators.py

```diff
@@ -2,19 +2,43 @@
 
 
 # Standard library
+import dataclasses as dc
 from typing import Any
-from typing import Callable
-from typing import Mapping
+from typing import Literal
 from typing import Optional
+from typing import Sequence
 
 # Third-party
 import numpy as np
 import xarray as xr
 
 
+@dc.dataclass
+class TargetCoordinatesAttrs:
+    """Attributes to the target coordinates."""
+
+    standard_name: str
+    long_name: str
+    units: str
+    positive: Literal["up", "down"]
```

Comment:
is there a  specific reason to not make this an enum?

Reply:
I chose the string literals as a trade off. Enums are stricter at runtime (provided you respect the type hint) but they require the caller to import an additional symbol at the call site. With the string literals, the syntax is lighter but some assistance is still provided by the type checker.